src/include/block.h

```diff
@@ -104,6 +104,18 @@ struct __wt_size {
     WT_SIZE *next[WT_SKIP_MAXDEPTH]; /* Size skiplist */
 };
 
+#ifdef HAVE_UNITTEST
+/*
+ * Per session handle cached block manager information.
+ */
+typedef struct {
```

Comment:
This should sit in the unit test code if it is only used there.

Reply:
Oh right, it's used all through block_session.c, in that case just drop the #ifdef HAVE_UNITTEST and move it entirely IMO.

Reply:
@jiechenbo can we action this one? It'll cleanup the PR a bit.

Reply:
Fixed up now